modules/react/collection/lib/ListBox.tsx

```diff
@@ -28,6 +29,8 @@ export const useListBox = createElemPropsHook(useListModel)(model => {
     style: {
       position: 'relative' as const,
       height: model.state.isVirtualized ? model.state.UNSTABLE_virtual.totalSize : undefined,
+      // marginTop: 0,
+      // marginBottom: 0,
```

Comment:
Do we need these removed or were these going to be used? Since their comments.

Reply:
fixed